deploy/chart/crds/0000_50_olm_00-catalogsources.crd.yaml

```diff
@@ -120,6 +120,9 @@ spec:
                   type: integer
                 publisher:
                   type: string
+                runAsRoot:
+                  description: RunAsRoot allows admins to indicate that they wish to run the CatalogSource pod in a privileged more as root
```

Comment:
```suggestion
                  description: RunAsRoot allows admins to indicate that they wish to run the CatalogSource pod in a privileged pod as root.  This should only be enabled when running older catalog images which could not be run as non-root.
```

Reply:
Has been incorporated in the API PR https://github.com/operator-framework/api/pull/258